public/js/Histogram.js

```diff
@@ -12,21 +16,28 @@ _.mixin(require('lodash-deep'));
  * @param {array} featureNames - An array of the features used in this screen.
  */
 function Histogram(sampleData, element, featureNames) {
+    var self = this;
```

Comment:
Why is this line needed? Why not just use this everywhere? (which you mostly do)